src/nerve/frontends/tui/commander/loop.py

```
@@ -0,0 +1,345 @@
+    commander.timeline.add(block)
+
+    # Render the running block
+    commander.timeline.render_last(commander.console)
+
+    # Execute
+    start_time = time.monotonic()
+
+    try:
+        commander._active_node_id = node_id
+        result = await commander._adapter.execute_on_node(node_id, expanded_prompt)
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)